.travis/deploy.sh

```diff
@@ -2,6 +2,16 @@
 
 # SCE is published to snowplow-maven which doesn't work with release-manager hence a specific script
 
+tag=$1
+project_version=$(sbt "show version" | tail -n 1 | awk '{print $2}')
+
+if [ "${project_version}" == "${tag}" ]; then
+    echo "Tag matches version of the project ('${project_version}'). Continuing with the deployment!"
+else
```

Comment:
I think that we can remove these lines (and change the `==` to `!=`)

Reply:
thanks @benjben , just addressed your feedback!